src/stashcache.py

```diff
@@ -0,0 +1,136 @@
+
+from collections import defaultdict
+import re
+import ldap
+import asn1
+import hashlib
+
+__oid_map = {
+   "DC": "0.9.2342.19200300.100.1.25",
+   "OU": "2.5.4.11",
+   "CN": "2.5.4.3",
+   "O": "2.5.4.10",
+   "ST": "2.5.4.8",
+   "C": "2.5.4.6",
+   "L": "2.5.4.7",
+   "postalCode": "2.5.4.17",
+   "street": "2.5.4.9",
+   "emailAddress": "1.2.840.113549.1.9.1",
+   }
+
+
+__dn_split_re = re.compile("/([A-Za-z]+)=")
+
+def _generate_ligo_dns():
```

Comment:
Not a big fan of having LIGO-specific code right inside the webapp -- I'd rather have some sort of configuration. If you can't fix it now, please make a JIRA ticket and assign it to me.

Reply:
Right - so this is basically grandfathered in until SciTokens is more mature.  How would you like to handle it?

Reply:
Decided to go ahead and make this a config knob.  Resolving for now.